### DISCUSSION TITLE
中文

### DISCUSSION THREAD

**Discussion by 0x9527-zhou** [community/community#20993] [Other Feature Feedback, Questions, & Ideas]
中文是世界上最有意思的语言，不接受反驳！

**Comment by dlvdls18** [comment]
### Translation

**Title**: Chinese
**Description**: Chinese is the most interesting language in the world, no refutation is accepted!

**Comment by yusancky** [comment]
**I also think that GitHub should support Chinese.** 
Now, somewhere such as [Github Docs](https://docs.github.com/cn) support Chinese.

**Comment by Matty25481** [comment]
I think all world languages should be accepted

**Reply by yusancky** [reply]
Great!
